src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -658,6 +657,12 @@ public void init() {
             mainStage.titleProperty().bind(windowTitle);
         });
         initShowTrackingNotification();
+        booleanBinding = Bindings.isEmpty(tabbedPane.getTabs());
```

Comment:
I would suggest renaming the variable to a more meaninful name .e.g noTabsBinding